torchbenchmark/util/extra_args.py

```diff
@@ -113,17 +119,28 @@ def apply_decoration_args(model: 'torchbenchmark.util.model.BenchmarkModel', dar
         # model handles amp itself if it has 'enable_amp' callback function (e.g. pytorch_unet)
         if hasattr(model, "enable_amp"):
             model.enable_amp()
-        elif model.test == "eval":
+        if not hasattr(model, "amp_context"):
```

Comment:
This is not needed, because it is suppose to be handled in `model.enable_amp()`.

Reply:
Done.